tasks/ceph_manager.py

```diff
@@ -727,11 +731,11 @@ def do_dump_ops(self):
             for osd in self.live_osds:
                 # Ignore errors because live_osds is in flux
                 self.ceph_manager.osd_admin_socket(osd, command=['dump_ops_in_flight'],
-                                     check_status=False, timeout=30)
+                                     check_status=False, timeout=30, stdout=DEVNULL)
```

Comment:
It is a thasher test to see if the OSD crashes or if there is no locking deadlock or something.  No code is examining the output for correctness.